client/src/components/Accordion/SingleAccordion/SingleAccordion.jsx

```diff
@@ -0,0 +1,47 @@
+import { React, useState, useRef, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faAngleDown } from '@fortawesome/free-solid-svg-icons';
+import './SingleAccordion.scss';
+
+const SingleAccordion = ({ header, children, toggle }) => {
```

Comment:
For initial value, pass in a prop `isInitiallyOpened`
and in the line below do  `const [isActive, setActive] = useState(isInitiallyOpened ? 'active' : 'closed');` for example
This initially opened might be needed for UX to show users that the accordion can be opened (by leaving one opened by default)